QRoute.xcodeproj/xcshareddata/xcschemes/QRoute-watchOS.xcscheme

```diff
@@ -38,8 +36,8 @@
             ReferencedContainer = "container:QRoute.xcodeproj">
          </BuildableReference>
       </MacroExpansion>
-      <AdditionalOptions>
-      </AdditionalOptions>
+      <Testables>
+      </Testables>
```

Comment:
Is this still useful?

Reply:
Not understanding your question?

Reply:
These are updates made by Xcode project.

Reply:
Xcode puts empty elements in the file, but it's not a concern.